format/view-spec.md

```diff
@@ -42,12 +42,28 @@ An atomic swap of one view metadata file for another provides the basis for maki
 
 Writers create view metadata files optimistically, assuming that the current metadata location will not be changed before the writer's commit. Once a writer has created an update, it commits by swapping the view's metadata file pointer from the base location to the new location.
 
+### Materialized Views
+
+Materialized views are a type of view with precomputed results from the view query stored as a table.
+When queried, engines may return the precomputed data for the materialized views, shifting the cost of query execution to the precomputation step.
+
+Iceberg materialized views are implemented as a combination of an Iceberg view and an underlying Iceberg table, the "storage-table", which stores the precomputed data.
+Materialized View metadata is a superset of View metadata with an additional pointer to the storage table and refresh metadata.
```

Comment:
```suggestion
Materialized View metadata is a superset of View metadata with an additional pointer to the storage table with refresh metadata.
```

Reply:
`and` would be misinterpreted that the refresh metadata is part of the view entity. `with` is more clear that the refresh state is part of the storage table